packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -34,14 +34,21 @@ export interface CSpellSettingsWithSourceTrace extends CSpellSettings {
 export interface FileSettings extends ExtendableSettings, CommandLineSettings {
     /**
      * Configuration format version of the settings file.
+     *
+     * This controls how the settings in the configuration file behave.
+     *
      * @default "0.2"
      */
     version?: Version;
 
     /** Words to add to global dictionary -- should only be in the user config file. */
     userWords?: string[];
 
-    /** Other settings files to be included. */
+    /**
+     * Allows this configuration to inherit configuration for one or more other files.
+     *
+     * See [the documentation](https://cspell.org/docs/configuration/imports/) for more details.
```

Comment:
The current link is: `[Importing / Extending Configuration](https://cspell.org/configuration/imports/)`

Reply:
can you offer this as a suggestion so i can click the button "commit suggestion"?